.circleci/config.yml

```
@@ -45,33 +28,22 @@ jobs:
 
   build_elixir_1_10_otp_23:
     docker:
-      - <<: *otp_23_image
-    environment:
-      ELIXIR_VERSION: 1.10.3-otp-23
-      LC_ALL: C.UTF-8
+      - image: hexpm/elixir:1.10.4-erlang-23.1.2-alpine-3.12.1
     <<: *defaults
     steps:
       - checkout
-      - <<: *install_elixir
       - <<: *install_hex_rebar
       - run: mix deps.get
       - run: mix test
       - run: mix archive.build
       - run: mix archive.install
-      - run: mix format --check-formatted
-      - run: mix docs
-      - run: mix hex.build
 
   build_elixir_1_9_otp_22:
     docker:
-      - <<: *otp_22_image
-    environment:
-      ELIXIR_VERSION: 1.9.4-otp-22
-      LC_ALL: C.UTF-8
+      - image: hexpm/elixir:1.9.4-erlang-22.3.4.9-alpine-3.12.0
```

```suggestion
      - image: hexpm/elixir:1.9.4-erlang-22.3.4.9-alpine-3.12.1
```

Should this also use the same alpine image?

---

That image doesn't exist. They don't build all combinations. It seems like whatever version of alpine was around when the elixir release was made is the one that was used.